### PR TITLE
fix(padding): fix padding variables in accordance with Frontify

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/theme/_variables.scss [hunk 1]
```diff
@@ -98,17 +98,11 @@ $icon-font-svg-id:        'glyphicons_halflingsregular' !default;
 //
 //## Define common padding and border radius sizes and more. Values based on 14px text and 1.428 line-height (~20px to start).
 
-$padding-base-vertical:     5px !default;
```
Comment (frassinier) [inline src/theme/_variables.scss]: These vars are used through Bootstrap, if you delete these lines then Bootstrap will use its default ones

Comment (mtikhinskaya) [inline src/theme/_variables.scss]: Moved to special section for rewritten variables

File: src/theme/_variables.scss [hunk 2]
```diff
@@ -877,3 +871,17 @@ $dl-horizontal-offset:        $component-offset-horizontal !default;
 $dl-horizontal-breakpoint:    $grid-float-breakpoint !default;
 //** Horizontal line color.
 $hr-border:                   $gray-lighter !default;
+
+
+//== Rewrite
+//
+//## This variables are used only for rewriting bootstrap variables
+
+$padding-base-vertical:     $padding-smaller;
```
Comment (frassinier) [inline src/theme/_variables.scss]: You must flag theses values as default in order to be overridden if needed
Could you please set them below Frontify ones to be used inside that file?

Comment (frassinier) [inline src/theme/_variables.scss]: What I mean is to copy block from line 880 to 112 in order to restore its position

**Pull request merged**